tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacade/BraintreeFacadePreCheckTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Braintree\Business\BraintreeFacade;
+
+use Braintree\Transaction\CreditCardDetails;
+use SprykerEco\Zed\Braintree\BraintreeConfig;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\AbstractTransaction;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\PreCheckTransaction;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeBridge;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeInterface;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Braintree
+ * @group Business
+ * @group Facade
+ * @group BraintreeFacadePreCheckTest
+ * Add your own group annotations below this line
+ */
+class BraintreeFacadePreCheckTest extends AbstractFacadeTest
+{
+    /**
+     * @return void
+     */
+    public function testPreCheckPaymentWithSuccessfulResponse(): void
+    {
+        // Arrange
+        $braintreeBusinessFactory = $this->getFactoryMock(['createPreCheckTransaction']);
+        $braintreeBusinessFactory->expects($this->once())->method('createPreCheckTransaction')->willReturn(
+            $this->getPreCheckTransactionMock(),
+        );
+        $braintreeFacade = $this->getBraintreeFacade($braintreeBusinessFactory);
+        $orderTransfer = $this->createOrderTransfer();
+        $quoteTransfer = $this->createQuoteTransfer($orderTransfer);
+
+        // Act
+        $braintreeTransactionResponseTransfer = $braintreeFacade->preCheckPayment($quoteTransfer);
+
+        // Assert
+        $this->assertTrue(
+            $braintreeTransactionResponseTransfer->getIsSuccess(),
+            'Facade should return success if the precheck payment transaction succeeded',
+        );
+    }
+
+    /**
+     * @return void
+     */
+    public function testPreCheckPaymentWithErrorResponse(): void
+    {
+        // Arrange
+        $factoryMock = $this->getFactoryMock(['createPreCheckTransaction']);
+        $factoryMock->expects($this->once())->method('createPreCheckTransaction')->willReturn(
+            $this->getPreCheckTransactionMock(false),
+        );
+        $braintreeFacade = $this->getBraintreeFacade($factoryMock);
+        $orderTransfer = $this->createOrderTransfer();
+        $quoteTransfer = $this->createQuoteTransfer($orderTransfer);
+
+        // Act
+        $response = $braintreeFacade->preCheckPayment($quoteTransfer);
```

Comment:
`$braintreeTransactionResponseTransfer`

Reply:
While github shows that this whole file as new, it was just moved from a different location. And this particular line was not affected by this PR. Should I apply this change anyway?

Reply:
As you suggested in a private discussion, I fixed this too.